test/docker/docker-compose.yml

```diff
@@ -25,6 +25,16 @@ services:
     ports:
       - "4500:4500"
 
+#docker run -p 8108:8108 -v/tmp/data:/data typesense/typesense:0.23.0 --data-dir /data --api-key=Hu52dwsas2AdxdE
+  tigris_search:
+    container_name: tigris_search
+    image: typesense/typesense:0.23.0
+    environment:
+      - TYPESENSE_DATA_DIR=/tmp
+      - TYPESENSE_API_KEY=Hu52dwsas2AdxdE
```

Comment:
should we find an alternate way to distribute this?

Reply:
This is for local dev, in prod we need a different key. We need to generate it and the pass it in the container.

Reply:
Should we make clear that it's not intended for production by just using something like 'ts_test_key1'?

Reply:
We need to set this key inside container which will be matched again the key that we are sending in the request. How will this work `ts_test_key1 `?

Reply:
We set it to `ts_test_key1` in both places.
Here for the `typesense` container and in our config for the requests we are sending.

Reply:
I tried it and it didn't work for me. There was some panic. I will debug it later.